controllers/account/ec2.go

```diff
@@ -628,8 +625,48 @@ func CreateEC2Instance(reqLogger logr.Logger, account *awsv1alpha1.Account, clie
 		}
 
 		// If fedramp, create subnet and set value for RunInstancesInput
-		if config.IsFedramp() {
-			subnetID, err := createSubnet(reqLogger, client, account, managedTags, customerTags, sampleCIDR, sampleVPCID)
+		if !config.IsFedramp() {
+			// Get a list of all VPCs with appropriate tag
+			result, err := client.DescribeVpcs(&ec2.DescribeVpcsInput{
+				MaxResults: aws.Int64(5),
+				Filters: []*ec2.Filter{
```

Comment:
Were you able to validate this query works for the fedramp VPC configuration?

Reply:
yeah I verified it